### PR TITLE
Add Minor modifications for benchmarks

### PR DESCRIPTION
**Pull request by Nirothipan**
## Approach
- GC logs are collected and the logs are analyzed using GC viewer https://github.com/chewiebug/GCViewer
- From the Analysis, the GC Throughput is extracted and written to the initial CSV file which contains the benchmarks results ( like function name, the time taken, avg latency etc ).

## Automation tests
 N/A

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? yes
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

### CODE REVIEW

**Comment by Nirothipan** [comment]
@chrishantha  Please review

File: benchmarks/executor [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+if [ ! -d "$resultsFolderName" ]; then
+ mkdir ${resultsFolderName}
+fi
+
+echo "Function_Name,Total Time (ms),Average Latency (ns),Throughput (operations/second),GC throughput (%),freedMemoryByFullGC (M),maxPause (s) " > ${resultsFileName}
+
+ballerina build benchmark
+
+for word in $(<benchmarkFunctions.txt)
```
Comment (chrishantha) [inline benchmarks/executor]: Rename "word" to a meaningful name.

File: benchmarks/pom.xml [hunk 2]
```diff
@@ -36,6 +36,11 @@
             <artifactId>ballerina</artifactId>
             <type>zip</type>
         </dependency>
+        <dependency>
+            <groupId>com.github.chewiebug</groupId>
+            <artifactId>gcviewer</artifactId>
```
Comment (chrishantha) [inline benchmarks/pom.xml]: Do not use this unless you have an approval from project leads.

File: benchmarks/README.md [hunk 3]
```diff
@@ -1,10 +1,8 @@
 # Ballerina-benchmarks
 
-### How To Visualize benchmarks 
+This module includes micro-benchmarks for ballerina.
```
Comment (chrishantha) [inline benchmarks/README.md]: s/ballerina/Ballerina

File: benchmarks/README.md [hunk 4]
```diff
@@ -1,10 +1,8 @@
...
 
-##### Prerequisite:
- Having the ballerina profile build.
- To build ballerina profile only `mvn clean install -P ballerina`
+### How To Visualize benchmarks
```
Comment (chrishantha) [inline benchmarks/README.md]: Reword this. We are not visualizing benchmarks here.

File: benchmarks/README.md [hunk 5]
```diff
@@ -1,10 +1,8 @@
...
-### How To Visualize benchmarks 
+This module includes micro-benchmarks for ballerina.
 
-##### Prerequisite:
- Having the ballerina profile build.
- To build ballerina profile only `mvn clean install -P ballerina`
+### How To Visualize benchmarks 
 
 ##### To run benchmarks: 
  `mvn clean install -P benchmarks -Dwarmup.iterations=<No_of_warmUp_Iterations> -Dbenchmark.iterations=<No_of_benchmark_Iterations>`
```
Comment (chrishantha) [inline benchmarks/README.md]: Use proper naming conventions. Use snake case properly.

File: benchmarks/pom.xml (hunk 2 above)
Comment (Nirothipan) [reply benchmarks/pom.xml]: Will finalize this after discussing.

**Pull request merged**